draft-ietf-httpbis-digest-headers.md

```diff
@@ -732,6 +774,14 @@ integrity at transport layer that protects HTTP header fields.
 
 A `Digest` header field using NOT RECOMMENDED digest-algorithms SHOULD NOT be used in signatures.
 
+## Usage with PATCH
+
+A `200 OK` response to a PATCH request would contain
+the `Digest` of the patched item, and the etag of the new object.
+
+This behavior is tighly coupled to the application logic
```

Comment:
I think this sentence comes from me on some earlier thread and I'm not really sure what you're aiming to achieve by including it in this context.

Lets figure out the core intent and write a clear message (I'll help), or just remove it because the new section "Use of Digest when acting on resources" does the job now.